src/enumerable.cr

```diff
@@ -984,6 +1019,43 @@ module Enumerable(T)
     max_by? &.itself
   end
 
+  # Returns an array of the maximum `k` elements, sorted descending.
```

Comment:
The docs do not mention what happens when `k` is larger than the number of elements (it simply returns all elements in that case)